src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -26,7 +26,7 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.IO.CSV.WriteToFile</oldName>
-    <newName>DSCore.ImportExport.ExportCSV</newName>
+    <newName>DSCore.ImportExport.Import.ExportCSV</newName>
```

Comment:
Yes, the task to move these methods were assigned to me and they were to be changed in https://github.com/DynamoDS/Dynamo/pull/7761. 
May I suggest keeping them as they are now? Though I think the correct method should be `DSCore.ImportExport.ExportCSV`, if the namespace is modified to `DSCore` and the class name is kept as `ImportExport`?